Exiled.Loader/ConfigManager.cs

```diff
@@ -14,38 +14,14 @@ namespace Exiled.Loader
     using Exiled.API.Extensions;
     using Exiled.API.Features;
     using Exiled.API.Interfaces;
-    using Exiled.Loader.Features.Configs;
 
     using YamlDotNet.Core;
-    using YamlDotNet.Serialization;
-    using YamlDotNet.Serialization.NamingConventions;
-    using YamlDotNet.Serialization.NodeDeserializers;
 
     /// <summary>
     /// Used to handle plugin configs.
     /// </summary>
     public static class ConfigManager
     {
-        /// <summary>
-        /// Gets the config serializer.
-        /// </summary>
-        public static ISerializer Serializer { get; } = new SerializerBuilder()
```

Comment:
You shouldn't completely remove Serializer and Deserializer, to not break plugins that use those two properties, you should "redirect" them to the moved Serializer and Deserializer.
Like `public static ISerializer Serializer => Loader.Serializer;`.